connect/transforms/src/main/java/org/apache/kafka/connect/transforms/Cast.java

```diff
@@ -39,12 +40,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.util.EnumSet;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Locale;
-import java.util.Map;
-import java.util.Set;
+import java.nio.ByteBuffer;
+import java.util.*;
```

Comment:
Wildcard imports should be caught by checkstyle, and should fail the build. In any case, please replace with non-wildcard imports.

Reply:
IntelliJ did this automatically for me - I have changed the code accordingly.